recipes/qt/5.x.x/conanfile.py

```diff
@@ -697,6 +699,8 @@ def build(self):
 
         args.append("-feature-gssapi" if self.options.get_safe("with_gssapi", False) else "-no-feature-gssapi")
 
+        args.append("-bundled-xcb-xinput")
```

Comment:
Can you give me a bit more insight into this one? I see the linked issue, but would appreciate more context for tracability's shake in the future :)

Reply:
Not specifying this argument allows qt to choose either the bundled version of xinput, either the system version. We cannot force to use system version because xenial does not have a package for it, so we force the bundled version (it makes the linked libraries coherent across distros)